app/actions.js

```diff
@@ -369,7 +370,8 @@ export const updateAllsChannelsUnreadCount = ({ addr, channelMessagesUnread }) =
   const allChannelsUnreadCount = Object.values(channelMessagesUnread).reduce((total, value) => {
     return total + (value || 0)
   }, 0)
-  if (allChannelsUnreadCount !== getState()?.cabals[addr]?.allChannelsUnreadCount) {
```

Comment:
I kinda like the ? syntax over the lodash get function. The get function is a little hard to read and not obvious as to what it does. What do you think?

Reply:
I agree, but the issue is that linter doesn't support that syntax yet.

Reply:
i don't think the code should become less readable because the linter complains, esp cause @feross will probably fix it sooner or later.

having a workflow of just running `standard --fix` (or `npm run lint`) after a PR has been reviewed solves any problems this git-hook introduces imo

also i really like that you fixed the linting issues in a separate PR! when it's done in a feature PR it becomes really hard to review :3

Reply:
@cblgh @nikolaiwarner  fixed

Reply:
nice!